js/assessments/paragraphTooShortAssessment.js

```diff
@@ -0,0 +1,87 @@
+var AssessmentResult = require( "../values/AssessmentResult.js" );
+var fixFloatingPoint = require( "../helpers/fixFloatingPoint.js" );
+var filter = require( "lodash/filter" );
+
+
+/**
+ * The function to use as a filter for too short paragraphs.
+ * @param {number} recommendedValue The recommended minimum length of a paragraph.
+ * @param {number} paragraphLength The number of words within a paragraph.
+ * @returns {boolean} Returns true if paragraphLength is lower than recommendedValue.
+ */
+var isParagraphTooShort = function( recommendedValue, paragraphLength ) {
+	return paragraphLength < recommendedValue;
+};
+
+/**
+ * Returns an array containing only the paragraphs shorter than the recommended length.
+ * @param {array} paragraphsLength The array containing the lengths of individual paragraphs.
+ * @param {number} recommendedValue The recommended minimum length of a paragraph.
+ * @returns {number} The number of too short paragraphs.
+ */
+var getTooShortParagraphs = function( paragraphsLength, recommendedValue ) {
+	var tooShortParagraphs = filter( paragraphsLength, isParagraphTooShort.bind( null, recommendedValue ) );
+	return tooShortParagraphs.length;
+};
+
+/**
+ * Returns the scores and text for the ParagraphTooShortAssessment
+ * @param {array} paragraphsLength The array containing the lengths of individual paragraphs.
+ * @param {number} tooShortParagraphs The number of too short paragraphs.
+ * @param {number} recommendedValue The recommended minimum length of a paragraph.
+ * @param {object} i18n The i18n object used for translations.
+ * @returns {{score: number, text: string }} the assessmentResult.
+ */
+var calculateParagraphLengthResult = function( paragraphsLength, tooShortParagraphs, recommendedValue, i18n ) {
+	// 6 is the number of scorepoints between 3, minscore and 9, maxscore. For scoring we use 40 steps, each step is 0.15
+	// Up to 13 is for scoring a 3, higher numbers give higher scores.
+	// floatingPointFix because of js rounding errors
+	var score = 3 + Math.max( Math.min( ( 0.15 ) * ( paragraphsLength[ 0 ] - 13 ), 6 ), 0 );
+	score = fixFloatingPoint( score );
+	if ( score >= 7 ) {
+		return {
+			score: score,
+			text: i18n.dgettext( "js-text-analysis", "None of your paragraphs is too short, which is great." )
```

Comment:
I'd vouch for replacing `is` with `are`. Both are grammatically correct, but seeing as we're talking about multiple paragraphs, `are` feels more natural.